pulpcore/cli/file/context.py

```diff
@@ -59,6 +59,7 @@ class PulpFilePublicationContext(PulpEntityContext):
     HREF = "file_file_publication_href"
     ID_PREFIX = "publications_file_file"
     CAPABILITIES = {"roles": [PluginRequirement("file", min="1.11.0")]}
+    NULLABLES = {"manifest"}
 
     def preprocess_entity(self, body: EntityDefinition, partial: bool = False) -> EntityDefinition:
         body = super().preprocess_entity(body, partial=partial)
```

Comment:
We could add an additional client side version check on `manifest is None` here to have validation before sending the body to the server.

Reply:
It didn't seem necessary IMO. Users get a user friendly message when they try to set manifest to null for older versions of pulp_file:

```
$ pulp file repository create --name test --manifest ""
Error: {"manifest":["This field may not be null."]}
```

Reply:
I agree. This was just recalling @ggainey 's mantra: "Whatever we can catch before bugging the server..."